README.md

```diff
@@ -106,6 +106,34 @@ buildctl build ... --exporter=local --exporter-opt output=path/to/output-dir
 buildctl du -v
 ```
 
+#### Running containerized buildkit
+
+buildkit daemon supports communicating with clients over TCP which makes it possible, among other things, to run buildkit within _Docker for Mac_. 
+
+To run daemon in a container:
+
+```
+$ docker run --rm --privileged -p 12345:12345 --tmpfs /tmp  buildkit:buildd-standalone buildd-standalone --addr tcp://0.0.0.0:12345 --root /tmp/buildkit
```

Comment:
@vdemeester @tonistiigi Please suggest (tentative) TCP port?

Reply:
@AkihiroSuda ahah I'm bad with numbers *and* names 😛 😅

Reply:
We haven't requested a fixed one (and probably shouldn't) so some incremental numbers or some round number sgtm for now.